src/KeyVault/KeyVault/Helpers/JwkHelper.cs

```diff
@@ -7,6 +7,21 @@ namespace Microsoft.Azure.Commands.KeyVault.Helpers
 {
     internal static class JwkHelper
     {
+        internal static RSACryptoServiceProvider ConvertToRSAKey(JsonWebKey jwk)
+        {
+            if (jwk == null || !jwk.Kty.Equals("RSA"))
+            {
+                return null;
+            }
+            var csp = new RSACryptoServiceProvider();
+            csp.ImportParameters(new RSAParameters()
```

Comment:
We need to catch the exception. If there is any error, let's return null. 

https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsacryptoserviceprovider.importparameters?view=net-5.0#System_Security_Cryptography_RSACryptoServiceProvider_ImportParameters_System_Security_Cryptography_RSAParameters_